examples/ticketer.py

```diff
@@ -951,6 +1190,10 @@ def run(self):
     else:
         password = options.password
 
+    if options.impersonate and not options.request:
```

Comment:
whenever options.impersonate == True , then several parameters should be passed ( request , aesKey, domain-sid, nthash ) in order to avoid the script crash. Somethig like :
```
missing_params = [ param_name for ( param, param_name ) in zip ( [self.request, self.aesKey, elf.domain_sid, self.nthash], [ "request","aesKey","domain-sid","nthash"] ) if not param is None ]
if missing_params:
    loggin.error("missing parameters to do sapphire ticket : ", ", ".join( missing_params ))
    sys.exit(1)
```

Reply:
same thing ( but little different) for _groups, extra-sid, duration,_ and any other parameter not being considered during sapphire ticket. It would be nice to inform the operator they won't be taken into account.

Reply:
Done, but there's an issue, the groups and duration have a default value. This means that even if a user doesn't supply the corresponding arguments, the message saying that the parameters will be ignored is still printed